src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -38,11 +40,31 @@ public static function dump($var)
         return call_user_func(self::$handler, $var);
     }
 
-    public static function setHandler(callable $callable = null)
+    /**
+     * @final since 4.1
+     */
+    public static function setHandler(callable $callable = null/*, bool $lock = false*/)/*: ?callable*/
     {
+        $lock = \func_num_args() > 1 ? func_get_arg(1) : false;
```

Comment:
I think we can to without this change: just call the method twice to restore

Reply:
Not sure to understand what you meant?

Reply:
I meant we should be able to make this work without changing this method at all.